### PR TITLE
Open source USB oscilloscope

### PR DESCRIPTION
**Pull request by elmot**

### CODE REVIEW

File: org/elmot/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+layout: org
+title: Elijah Motornyy(http://elmot.xyz)
```
Comment (Arachnid) [inline org/elmot/index.md]: You should leave the URL out of the title; it'll show up as a prefix to all PIDs under this org.

File: 1209/e116/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Open-Source Oscilloscope based on STM32F3-DISCOVERY board
```
Comment (Arachnid) [inline 1209/e116/index.md]: Does the project have a (shorter) name of its own?

**Comment by Arachnid** [comment]
Can you describe what you need the PID for here? It looks like you're using stock hardware for the USB interface.

**Comment by elmot** [comment]
Fixed

**Comment by elmot** [comment]
Well, I use a stock prototype board, but it has only a physical  USB interface but no VID/PID. The on-board chip is shipped empty, firmware is completely mine, and, unfortunately, st.com does not provide any PIDs for products based on their chips or proto boards. And that's a reason why I request PID here.

**Comment by elmot** [comment]
Thank you a lot.
great job!

**Pull request merged**